### PR TITLE
Add simple batch decomposition in DataRouter

### PR DESCRIPTION
**Pull request by esavier**
I would like to send multiple cdl messages inside one MQ message,
Messages should look like this:
```
(pseudocode)
[ 
  {id:1,schema:1,data:{}}
  {id:2,schema:1,data:{}}
  {id:3,schema:1,data:{}}
  {id:4,schema:2,data:{}}
  {id:5,schema:3,data:{}}
]
```
decomposition should in this case look (at this point) as 5 separate messages containing one message each, without array

### CODE REVIEW

File: data-router/src/main.rs [hunk 1]
```diff
@@ -91,22 +92,46 @@ async fn handle_message(
...
+                let data = CommandServiceInsertMessage {
+                    object_id: event.object_id,
+                    payload: event.data,
+                    schema_id: event.schema_id,
+                    timestamp: since_the_epoch.as_millis() as i64, // TODO: Change types?
+                };
+
+                let payload = serde_json::to_vec(&data).unwrap_or_default();
+                let key = data.object_id.to_string();
+                send_message(producer.as_ref(), &topic_name, &key, payload).await;
```
Comment (Kononnable) [inline data-router/src/main.rs:119]: Can we extract this logic to some function and use it on both sides of `if` statement?

File: data-router/src/main.rs [hunk 2]
```diff
@@ -91,22 +92,46 @@ async fn handle_message(
...
+            }
+        } else {
+            let event  = serde_json::from_str(message.payload()?).context("Payload deserialization failed")?;
```
Comment (Kononnable) [inline data-router/src/main.rs:122]: Already done in line 104

File: data-router/src/main.rs (hunk 1 above)
Comment (esavier) [reply data-router/src/main.rs:119]: yes, doing it now

File: data-router/src/main.rs (hunk 2 above)
Comment (esavier) [reply data-router/src/main.rs:122]: making it look "nice" is too much of a hassle, without much gain, also i personally relly dont like enum variants (and later enum recognition) which is one of the ways to do it.

**Comment by esavier** [comment]
starting from scratch will be easier than merging at this point

**Pull request closed without merging**